src/components/Toast.js

```diff
@@ -1,6 +1,6 @@
 import { classNames } from "@/utils/classNames";
 
-export default function Toast({ type = "info", message }) {
+export default function Toast({ type = "info", message, repo }) {
```

Comment:
The "repo" param contains the details of repo like owner and repo name.

Reply:
For adding styling to the Repo name, I have to create a new Param "repo".
![Screenshot 2024-01-07 103210](https://github.com/EddieHubCommunity/RepoRater/assets/124811276/ef7af927-9bd3-4ab1-aea7-e764f7589be1)

Reply:
Link is corrected.
![Uploading Screenshot 2024-01-07 105338.png…]()

Reply:
Valid link:
![Screenshot 2024-01-07 105338](https://github.com/EddieHubCommunity/RepoRater/assets/124811276/b7d6228e-6c25-4853-9126-ed54f0d2a281)

Alert:
![Screenshot 2024-01-07 110030](https://github.com/EddieHubCommunity/RepoRater/assets/124811276/525ceed6-7bac-4892-a795-792cd6a76624)

Reply:
> @amanr-dev Thank you for the clarification. It would be beneficial to document the structure of the "repo" parameter, indicating that it includes both the owner and repository name, to provide clear expectations for future contributors and maintainers. Additionally, consider adding type-checking to ensure that the object passed has the necessary properties.
> 
> New Learnings Added


But the files do not uses type checking and propType

Reply:
Ok